setup.py

```diff
@@ -122,6 +122,17 @@
     author_email='rbean@redhat.com',
     url='https://github.com/fedora-infra/fedmsg/',
     license='LGPLv2+',
+    classifiers=[  # https://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 3 - Alpha',
```

Comment:
> All these justifications assume that fedmsg follows semantic versioning 2.0.0 (http://semver.org/). @ralphbean, since you're the original author, is this assumption false?

In theory, no - the assumption is correct.  In practice, yes - the assumption is incorrect.

Like @pypingou pointed out, there are more than a few apps depending on the existing API regardless of whether or not a `1.x` promise was made.

I'd advise treating `0.x` as a stable line and any API breaking changes could go into a new `1.x` line.  You could even jump straight from `0.x` to `2.x` to make the jump obvious to users.